pom.xml

```diff
@@ -143,11 +149,18 @@
                         </goals>
                     </execution>
                     <execution>
+                        <id>scala-test-compile</id>
+                        <phase>process-test-resources</phase>
                         <goals>
-                            <goal>compile</goal>
                             <goal>testCompile</goal>
                         </goals>
                     </execution>
+                    <execution>
+                        <id>attach-javadocs</id>
```

Comment:
Where is this coming from?

Reply:
It was taken from the other `scala-maven-plugin` config, which this deduplicates with `maven-scala-plugin` (which is the same).